mailbag/formats/pst.py

```diff
@@ -0,0 +1,72 @@
+from mailbag.email_account import EmailAccount
+from mailbag.models import Email
+import mailbox
+import pypff
+
+
+class PST(EmailAccount):
+    # pst - This concrete class parses PST file format
+    format_name = 'pst'
+
+    def __init__(self, target_account, **kwargs):
+        print("Parsity parse")
+        # code goes here to set up mailbox and pull out any relevant account_data
+
+
+        self.file =target_account.input_path
+        print("Reading :", self.file)
+
+
+
+
+
+
+
+    def account_data(self):
+        return account_data
+
+    def parse_folder(root):
+        messages=[]
+
+        for folder in root.sub_folders:
+            if folder.number_of_sub_folders:
+                messages += PST.parse_folder(folder)
+
+
+            for message in folder.sub_messages:
+                messages.append({
```

Comment:
I believe that appending each message to `messages` may avoid the performance benefits of using a generator, right? So this should either be a generator as well or be combined with the messages generator below.